Content.Client/Traits/ParacusiaSystem.cs

```diff
@@ -0,0 +1,73 @@
+using Content.Shared.Traits.Assorted;
+using Content.Client.Camera;
+using Robust.Shared.Random;
+using Robust.Client.GameObjects;
+using Robust.Client.Player;
+using Robust.Shared.Timing;
+
+namespace Content.Client.Traits;
+
+public sealed class ParacusiaSystem : SharedParacusiaSystem
+{
+    [Dependency] private readonly IGameTiming _timing = default!;
+    [Dependency] private readonly IPlayerManager _player = default!;
+    [Dependency] private readonly IRobustRandom _random = default!;
+    [Dependency] private readonly SharedAudioSystem _audio = default!;
+    [Dependency] private readonly CameraRecoilSystem _camera = default!;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+        SubscribeLocalEvent<ParacusiaComponent, ComponentStartup>(SetupParacusia);
+        SubscribeLocalEvent<ParacusiaComponent, PlayerDetachedEvent>(OnPlayerDetach);
+    }
+
+    private void OnPlayerDetach(EntityUid uid, ParacusiaComponent component, PlayerDetachedEvent args)
+    {
+        component.Stream?.Stop();
+    }
+
+    private void SetupParacusia(EntityUid uid, ParacusiaComponent component, ComponentStartup args)
+    {
+        component.NextIncidentTime = _timing.CurTime +
+                                     TimeSpan.FromSeconds(_random.NextFloat(component.MinTimeBetweenIncidents, component.MaxTimeBetweenIncidents));
+    }
+
+    public override void Update(float frameTime)
+    {
+        base.Update(frameTime);
+
+        if (!_timing.IsFirstTimePredicted)
+            return;
+
+        var curTime = _timing.CurTime;
+        var localPlayer = _player.LocalPlayer?.ControlledEntity;
+
+        if (!TryComp<ParacusiaComponent>(localPlayer, out var paracusia))
+            return;
+
+        if (curTime < paracusia.NextIncidentTime)
+            return;
+
+        // Set the new time.
+        paracusia.NextIncidentTime = _timing.CurTime +
```

Comment:
same

Reply:
tried to do the same thing for this, trait did not work afterwards